static/js/components/UserChip.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import React from 'react';
+import { Card } from 'react-mdl/lib/Card';
+import Icon from 'react-mdl/lib/Icon';
+
+import { getPreferredName, getEmployer } from '../util/util';
+import ProfileImage from './ProfileImage';
+import { mstr } from '../util/sanctuary';
+import type { Profile } from '../flow/profileTypes';
+
+const UserChip = ({ profile }: {profile: Profile}): React$Element<*> => (
+  <Card className="user-chip" shadow={2}>
+    <div className="profile-info">
+      <span className="name">
+        { getPreferredName(profile) }
+      </span>
+      <span className="employer">
+        { mstr(getEmployer(profile)) }
+      </span>
+      <a href={`/users/${profile.username}`} className="mm-minor-action">
+        <Icon name="person" />
```

Comment:
when you hover over this link, an underline appears under this icon. looks kinda wonky. might be able to be solved in CSS?